src/components/PostEditor/PostEditor.store.js

```diff
@@ -93,23 +97,6 @@ export const getLinkPreview = ormCreateSelector(
     LinkPreview.idExists(linkPreviewId) ? LinkPreview.withId(linkPreviewId).ref : null
 )
 
-const getDefaultTopicsResults = makeGetQueryResults(FETCH_DEFAULT_TOPICS)
-
-export const getDefaultTopics = ormCreateSelector(
```

Comment:
again why remove the default topics stuff? I think we would want to display them by default when you start typing a #. Maybe. Not super important though

Reply:
I really don't think the behavior changed, did you experience something different in use? It was a code organization naming thing only if I'm remembering right...